scripts/ecs-anywhere-install.sh

```diff
@@ -307,6 +308,7 @@ install-ssm-agent() {
     else
         local dir
         dir="$(mktemp -d)"
+        chmod 755 "$dir"
```

Comment:
this could be moved down to only happen just before `dpkg -i` is run so that it doesn't affect rpms unecessarily, making it recursive might remove the need for the umask change.

Reply:
I'm indifferent - I see no harm in doing this, but if you think it's important, then we can be more precise.